bigbluebutton-html5/imports/ui/components/chat/component.jsx

```diff
@@ -56,10 +57,18 @@ const Chat = (props) => {
             <Link
               to="/users"
               role="button"
-              className={styles.closeIcon}
               aria-label={intl.formatMessage(intlMessages.closeChatLabel, { 0: title })}
+              tabIndex={-1}
             >
-              <Icon iconName="close" onClick={() => actions.handleClosePrivateChat(chatID)} />
+              <Button
+                className={styles.closeBtn}
+                label={intl.formatMessage(intlMessages.closeChatLabel, { 0: title })}
+                icon="close"
+                size="md"
+                hideLabel
+                onClick={() => actions.handleClosePrivateChat(chatID)}
+                aria-label={intl.formatMessage(intlMessages.closeChatLabel, { 0: title })}
```

Comment:
Why we duplicate the label in three places? Do we actually need them? A single one in the element that is being focused should satisfy.